src/modules/ShortcutGuide/ShortcutGuide/shortcut_guide.cpp

```diff
@@ -102,13 +102,45 @@ namespace
     const LPARAM eventActivateWindow = 1;
 }
 
+constexpr int alternative_switch_hotkey_id = 0x2;
+constexpr UINT alternative_switch_modifier_mask = MOD_WIN | MOD_SHIFT;
+constexpr UINT alternative_switch_vk_code = VK_OEM_2;
+
 OverlayWindow::OverlayWindow()
 {
+    instance = this;
     app_name = GET_RESOURCE_STRING(IDS_SHORTCUT_GUIDE);
     app_key = ShortcutGuideConstants::ModuleKey;
 
     Logger::info("Overlay Window is creating");
     init_settings();
+
+    auto switcher = [&](HWND hwnd, UINT msg, WPARAM wparam, LPARAM lparam) -> LRESULT {
```

Comment:
We will improve the code moving this to the WinProc function instead of passing this function, but we can do it in another PR.